build-packages/create-magento-app/index.js

```diff
@@ -8,6 +8,7 @@ const logger = require('@scandipwa/scandipwa-dev-utils/logger');
 const createFilesystem = require('@scandipwa/scandipwa-dev-utils/create-filesystem');
 const shouldUseYarn = require('@scandipwa/scandipwa-dev-utils/should-use-yarn');
 const installDeps = require('@scandipwa/scandipwa-dev-utils/install-deps');
+const googleAnalytics = require('@scandipwa/scandipwa-dev-utils/analytics');
```

Comment:
I don't see an `analytics` util in the dev-utils package, probably it was added in the newer version and at first glance, looks like you forgot to update the `package.json` for that :thinking:

Reply:
I assume when this one https://github.com/scandipwa/create-scandipwa-app/pull/50  will be merge we can bump dev-utils version in dependencies here